flexmeasures/cli/data_show.py

```diff
@@ -201,6 +206,25 @@ def show_generic_asset(asset):
     ]
     click.echo(tabulate(asset_data, headers=["Type", "Location", "Attributes"]))
 
+    child_asset_data = [
```

Comment:
Please add the Id and name of the child assets here.

And we should leave out location and attributes, actually (gets too wide otherwise, and people can easily get into the details with an explicit `flexmeasures show asset` call if they are interested.